pkg/api/handlers.go

```diff
@@ -161,6 +162,27 @@ func DefaultConfigHandler(actualCfg interface{}, defaultCfg interface{}) http.Ha
 	}
 }
 
+type configResponse struct {
+	Status string `json:"status"`
+	Config string `json:"data"`
+}
+
+func (cfg *Config) statusConfigHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		data, err := yaml.Marshal(cfg)
```

Comment:
I don't think `cfg` is what you want to marshal here. `cfg` is the configuration for the `API` struct itself. This method will need to marshal `actualConfig` like the `/config` endpoint does.

Reply:
Fixed. Thanks!